keyboards/rabbit/rabbit68/readme.md

```diff
@@ -0,0 +1,20 @@
+Rabbit 68
+===
+
+![Rabbit 68 Project](https://github.com/kaiec/rabbit68)
+
+A compact 68 keys ortholinear keyboard.
+
+Keyboard Maintainer: [Kai Eckert](https://github.com/kaiec)  
+Hardware Supported: Rabbit68 PCB rev1 
```

Comment:
```suggestion
Hardware Supported: Rabbit68 PCB rev1  
```

The two spaces at the end of these first two lines are necessary for GitHub to render the section properly.